homeassistant/components/youless/strings.json

```diff
@@ -14,5 +14,42 @@
     "error": {
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "total_water": {
+        "name": "Total water usage"
+      },
+      "total_gas_m3": {
+        "name": "Total gas usage"
+      },
+      "active_power_w": {
+        "name": "Current power usage"
+      },
+      "active_power_phase_w": {
+        "name": "Power phase {phase}"
+      },
+      "active_voltage_phase_v": {
+        "name": "Voltage phase {phase}"
+      },
+      "active_current_phase_a": {
+        "name": "Current phase {phase}"
+      },
+      "total_energy_import_tariff_kwh": {
+        "name": "Energy import tariff {tariff}"
+      },
+      "total_energy_import_kwh": {
+        "name": "Total energy import"
+      },
+      "total_energy_export_tariff_kwh": {
+        "name": "Energy export tariff {tariff}"
+      },
+      "total_s0_kwh": {
+        "name": "Total energy S0"
+      },
+      "active_s0_w": {
+        "name": "Current usage S0"
```

Comment:
what's s0?

Reply:
There is an additional port on the device called the S0 port. The official documentation mentions this has a current watt consumed sensor and an total increasing kwh sensor. What they measure depends on what you connect them to.

Reply:
Is this a common name users can recognise?

Reply:
I would hope so this is the name YouLess itself uses for the sensors. So if you look at the UI of the device it is also called S0.